octobot_trading/exchanges/connectors/ccxt/ccxt_adapter.py

```diff
@@ -33,6 +33,12 @@ def fix_order(self, raw, **kwargs):
             exchange_timestamp = fixed[ecoc.TIMESTAMP.value]
             fixed[ecoc.TIMESTAMP.value] = \
                 self.get_uniformized_timestamp(exchange_timestamp)
+            try:
+                fees = fixed[enums.ExchangeConstantsOrderColumns.FEE.value]
+                if fees is not None and fees[enums.ExchangeConstantsOrderColumns.COST.value] is None:
+                    fees[enums.ExchangeConstantsOrderColumns.COST.value] = 0
```

Comment:
I don't understand why we should set COST to 0 when fee is not None

Reply:
Otherwise we keep cost at None and it's creating issues later on. Do you think we should leave it at None and add if statements later on ?